### PR TITLE
Block Entity

### PR DESCRIPTION
**Pull request by DanyGames2014**

### CODE REVIEW

File: mappings/net/minecraft/entity/SpawnGroup.mapping [hunk 1]
```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_238 net/minecraft/entity/SpawnGroup
+	FIELD field_942 MONSTER Lnet/minecraft/class_238;
+	FIELD field_943 CREATURE Lnet/minecraft/class_238;
+	FIELD field_944 WATER_CREATURE Lnet/minecraft/class_238;
+	FIELD field_945 creatureClass Ljava/lang/Class;
+	FIELD field_946 capacity I
+	FIELD field_947 spawnMaterial Lnet/minecraft/class_15;
+	FIELD field_948 peaceful Z
+	METHOD <init> (Ljava/lang/String;ILjava/lang/Class;ILnet/minecraft/class_15;Z)V
+		ARG 3 creatrueClass
```
Comment (MartinSVK12) [inline mappings/net/minecraft/entity/SpawnGroup.mapping]: typo lol

Comment (DanyGames2014) [reply mappings/net/minecraft/entity/SpawnGroup.mapping]: thank you, fixed

**Review by huedev — APPROVED** [review]
Looks good, matches up with Yarn mappings where possible and good names for everything that does not have a direct Yarn equivalent.

**Review by Jadestrouble — APPROVED** [review]
looks good to me. dont really have any feedback as i haven't used these and im not familiar but it makes sense to me.

File: mappings/net/minecraft/block/entity/BlockEntity.mapping [hunk 2]
```diff
@@ -1,10 +1,27 @@
 CLASS net/minecraft/class_55 net/minecraft/block/entity/BlockEntity
+	FIELD field_1236 idToClass Ljava/util/Map;
+	FIELD field_1237 classToId Ljava/util/Map;
 	FIELD field_1238 world Lnet/minecraft/class_18;
 	FIELD field_1239 x I
 	FIELD field_1240 y I
 	FIELD field_1241 z I
+	FIELD field_1242 removed Z
+	METHOD method_1067 create (Ljava/lang/Class;Ljava/lang/String;)V
```
Comment (calmilamsy) [inline mappings/net/minecraft/block/entity/BlockEntity.mapping:9]: I don't like this name, but it's what yarn uses, so eh.

**Review by calmilamsy — APPROVED** [review]
Yeah, I see no issues.

**Pull request merged**